ec2/utility.py

```diff
@@ -0,0 +1,134 @@
+########
+# Copyright (c) 2014 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#    * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    * See the License for the specific language governing permissions and
+#    * limitations under the License.
+
+# built-in package
+import time
+
+# other packages
+from boto.ec2 import EC2Connection as EC2
+from boto.exception import EC2ResponseError, BotoServerError
+
+# ctx packages
+from cloudify import ctx
+from cloudify.exceptions import NonRecoverableError
+from cloudify.decorators import operation
+
+
+@operation
```

Comment:
this function and all of the below ones are not operations.
you should read what an operation is in the documentation.
these are all used by operations but are not operations themselves.